src/ProjectTemplates/Web.ProjectTemplates/RazorClassLibrary-CSharp.csproj.in

```diff
@@ -1,12 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework Condition="'$(NoPagesAndViews)' != 'True'">netcoreapp3.0</TargetFramework>
+    <TargetFramework Condition="'$(NoPagesAndViews)' == 'True'">netstandard2.0</TargetFramework>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.RazorClassLibrary1</RootNamespace>
   </PropertyGroup>
 
```

Comment:
I know this is super-nitpicky, but with the conditionals defined like this, will we get double blank lines in the resulting project files? It looks like blank lines 8 and 12 will always be included. To fix this, maybe delete blank line 12.